source/Angle.h

```diff
@@ -56,10 +57,18 @@ class Angle {
 	Point Unit() const;
 	// Convert an Angle object to degrees, in the range -180 to 180.
 	double Degrees() const;
+	// Convert an Angle object to degrees, in the range 0 to 360.
+	double AbsDegrees() const;
 
 	// Return a point rotated by this angle around (0, 0).
 	Point Rotate(const Point &point) const;
 
+	// Judge whether this is inside from "base" to "limit."
+	// The range from "base" to "limit" is expressed by "clock" orientation.
```

Comment:
normally in maths you would do counter clockwise, no?

Reply:
Idk. I think clockwise is more intuitive.